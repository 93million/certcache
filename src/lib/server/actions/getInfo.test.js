/* global test expect */

const getInfo = require('./getInfo')

test(
  'should output data to match snapshot',
  async () => {
    await expect(getInfo()).resolves.toMatchSnapshot()
  }
)
