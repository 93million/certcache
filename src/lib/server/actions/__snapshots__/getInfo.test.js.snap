// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should output data to match snapshot 1`] = `
Object {
  "extensions": Array [
    "certbot",
    "thirdparty",
  ],
  "version": "0.4.0-beta.0",
}
`;
